hdf/src/tbbt.c

```diff
@@ -15,15 +15,15 @@
 /* Extended from (added threads to) Knuth 6.2.3, Algorithm A (AVL trees) */
 /* Basic tree structure by Adel'son-Vel'skii and Landis */
 
-#include <stdio.h> /* NULL */
+#include <stdio.h>
+
 #include "hdf.h"
+
 #define TBBT_INTERNALS
 #include "tbbt.h"
-#define Alloc(cnt, typ) (typ *)HDmalloc((cnt) * sizeof(typ))
-#define Free(x)         (HDfree((VOIDP)x))
 
 #define KEYcmp(k1, k2, a)                                                                                    \
-    ((NULL != compar) ? (*compar)(k1, k2, a) : HDmemcmp(k1, k2, 0 < (a) ? (a) : (intn)HDstrlen(k1)))
+    ((NULL != compar) ? (*compar)(k1, k2, a) : memcmp(k1, k2, 0 < (a) ? (a) : (intn)HDstrlen(k1)))
 
 VOID tbbt1dump(TBBT_NODE *node, intn method);
```

Comment:
Is this VOID macro still necessary?

Reply:
I plan to remove it and its friend VOIDP.